src/explore-education-statistics-admin/src/pages/users/UserInvitePage.tsx

```diff
@@ -91,71 +96,122 @@ const UserInvitePage = ({
           </RelatedInformation>
         </div>
       </div>
-      <div className="govuk-grid-row">
-        <div className="govuk-grid-column-two-thirds">
-          {model && (
-            <Formik<FormValues>
-              enableReinitialize
-              initialValues={{
-                userEmail: '',
-                selectedRoleId:
-                  orderBy(model.roles, role => role.name)?.[0]?.id ?? '',
-              }}
-              validationSchema={Yup.object({
-                userEmail: Yup.string()
-                  .required('Provide the users email')
-                  .email('Provide a valid email address'),
-                selectedRoleId: Yup.string().required(
-                  'Choose role for the user',
-                ),
-              })}
-              onSubmit={handleSubmit}
-              render={_ => {
-                return (
-                  <Form id={formId}>
-                    <FormFieldset
-                      id={`${formId}-userEmailFieldset`}
-                      legend="Provide the email address for the user"
-                      legendSize="m"
-                      hint="The invited user must have a @education.gov.uk email address"
-                    >
-                      <FormFieldTextInput
-                        id={`${formId}-userEmail`}
-                        label="User email"
-                        name="userEmail"
-                      />
-                    </FormFieldset>
-
-                    <FormFieldset
-                      id={`${formId}-selectedRoleIdFieldset`}
-                      legend="Select the role of the user within the service"
-                      legendSize="m"
-                      hint="Most users should be added with the analyst role"
-                    >
+      {model && (
+        <Formik<FormValues>
+          enableReinitialize
+          initialValues={{
+            userEmail: '',
+            selectedRoleId:
+              orderBy(model.roles, role => role.name)?.[0]?.id ?? '',
```

Comment:
Should we be defaulting to the first role? I would recommend leaving this empty to make sure the user is aware of what they're selecting

Reply:
Yeah the first role is the most applicable (we only have two) 99% of the time a user will be added as an analyst which is what this defaults too. Saves the user an extra step.